caom2-artifact-resolvers/src/test/java/ca/nrc/cadc/caom2/artifact/resolvers/ResolverCapabilitiesMock.java

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
Maybe the slack discussion was not clear. I said to not try to mock the capabilities and just rely on production registry lookup to work. I was going to question why you implemented a separate 2-line getServiceURL method in CadcResolver, but since you did:

That method hides all the registry interaction and you could just bypass registry lookup with something like:
```
class TestResolver extends CadcResolver {
    @Override
    public URL getServiceURL() {
        return dummyURL;
    }
}```
This is a simple way to bypass code that is outside the scope you are trying to test.

Reply:
Needed the same thing in the cutout resolver test and did it there but forgot to redo it here. Done now.